py/server/tests/test_rollup_tree_table.py

```diff
@@ -0,0 +1,73 @@
+#
+#     Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+import unittest
+
+from deephaven import read_csv, empty_table
+from deephaven.agg import sum_, avg, count_, first, last, max_, min_, std, abs_sum, \
+    var
+from deephaven.filters import Filter
+from deephaven.rollup_table import NodeType
+from tests.testbase import BaseTestCase
+
+
+class RollupAndTreeTableTestCase(BaseTestCase):
```

Comment:
In my quick look here, it looks like the code coverage is very low.

Reply:
Actually all the supported operations and public methods are covered in the tests.